subnet/subnet.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 type LeaseAttrs struct {
-	PublicIP    ip.IP4
+	InterfaceIP ip.IP4
```

Comment:
Since `InterfaceIP` is being used on backends, isn't better to add only `json:"-"`?
As I said, it is the first time I am developing on `go`, and as far as I had understood if I remove this line it will complain. Or am I wrong?